src/stores/useBankStore.ts

```
@@ -25,12 +29,32 @@ export const useBankStore = defineStore('bankstore', {
-          if (res.amount) this.supply = res.amount;
+        // Fetch supply and burn address balance in parallel
+        Promise.all([
+          this.blockchain.rpc.getBankSupplyByDenom(denom),
+          this.blockchain.rpc.getBankBalanceByDenom(BURN_ADDRESS, denom)
+        ]).then(([supplyRes, burnBalanceRes]) => {
+          if (supplyRes.amount) {
+            // Store the original supply
+            const originalSupply = {...supplyRes.amount};
+
+            // Deduct burn address balance from total supply
+            if (burnBalanceRes && burnBalanceRes.balance) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)